CONTRIBUTING.md

```diff
@@ -98,6 +98,17 @@ java:path/to/java/file.java@Region
 kotlin:path/to/kotlin/file.kt@Region
 ```
 ~~~
+If you write java code, the wiki has a tool to generate translations from quilt mappings to mojmaps; simply run the
+`translateToMojmaps` task to generate translated code of any code in `src/main/java` of every project. To include these
+translations in the displayed code on a page, use the following syntax:
+~~~markdown
+```tabbed-files
+java:src/main/java/package/File.java@Region
+java@mojmaps:src_mojmaps/main/java/package/File.java@Region
+```
+~~~
+Note that this can be used for code chunks that are available only in one language - if only a single tab would be displayed,
+all tabs are hidden from a given code block.
```

Comment:
This should probably include that the default syntax is `qm` and what mapping sets and languages are supported for remapping and correct display in the wiki.